common/src/main/java/me/shedaniel/architectury/registry/trade/TradeRegistry.java

```diff
@@ -23,7 +23,19 @@
 import net.minecraft.world.entity.npc.VillagerProfession;
 import net.minecraft.world.entity.npc.VillagerTrades;
 
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
 public class TradeRegistry {
+    private static final List<Consumer<VillagerTradeOfferContext>> VILLAGER_MODIFY_HANDLERS = new ArrayList<>();
+    private static final List<Predicate<VillagerTradeOfferContext>> VILLAGER_REMOVE_HANDLERS = new ArrayList<>();
+    private static final List<Consumer<WanderingTraderOfferContext>> WANDERING_TRADER_MODIFY_HANDLERS = new ArrayList<>();
+    private static final List<Predicate<WanderingTraderOfferContext>> WANDERING_TRADER_REMOVE_HANDLERS = new ArrayList<>();
+    
+    private static final Map<VillagerProfession, Map<Integer, Integer>> VILLAGER_MAX_OFFER_OVERRIDES = new HashMap<>();
+    private static Integer WANDERING_TRADER_MAX_OFFER_OVERRIDE = null;
```

Comment:
wrong case

Reply:
just for `WANDERING_TRADER_MAX_OFFER_OVERRIDE` here right?

Reply:
yeah because it is not static final